examples/rbac/client/feature_repo/test_admin.py

```diff
@@ -0,0 +1,120 @@
+import os
+from datetime import datetime
+
+import pandas as pd
+from feast import FeatureStore
+from feast.data_source import PushMode
+
+
+def run_demo():
+    print("\n--- This is Admin Example ---")
+    try:
+        os.environ["LOCAL_K8S_TOKEN"] = ""
```

Comment:
I think this here setting token according to the server account in each file if the user setting in environment variable how the different sa token will be tested it may cause some confusion 
I would keep the file base token from pod priority incase if token regenerated.

Reply:
as you prefer, but usually env vars take priority to programmatic options. anyway it's not a real issue, apart from the fact that the user has to change the code instead of changing the env var to apply a new token 🤷‍♂️